InstrumentDB/0 Database.sql

```diff
@@ -0,0 +1,8 @@
+use master
```

Comment:
-- LL 100% Great Job!

Reply:
I corrected everything, committed and pushed.
Please let me know what's next.